salt/tests/unit/modules/test_metalk8s_grafana.yaml

```diff
@@ -0,0 +1,141 @@
+- dashboard: null
```

Comment:
details: I would rather put this file in a sub-directory (e.g. `golden_files`) to only have `.py` files at this level.

Reply:
True, I added `files` directory to be consistent we test we already have